composer.json

```diff
@@ -30,13 +30,24 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
-        "zendframework/zend-mail": "~2.0",
-        "zendframework/zend-http": "~2.0"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-mail": "^2.6",
+        "zendframework/zend-http": "^2.5",
+        "zendframework/zend-mime": "^2.6"
```

Comment:
`"zendframework/zend-servicemanager": "^2.7 || ^3.1"` should be added here as well